penglai-enclave-driver/penglai-enclave-ioctl.c

```diff
@@ -95,7 +95,8 @@ int penglai_enclave_create(struct file * filep, unsigned long args)
 {
 	struct penglai_enclave_user_param* enclave_param = (struct penglai_enclave_user_param*)args;
 	void *elf_ptr = (void*)enclave_param->elf_ptr;
-	long elf_size = enclave_param->elf_size;
+	int elf_size = 0;
+	penglai_enclave_elfmemsize(elf_ptr, &elf_size);
```

Comment:
The `penglai_enclave_elfmemsize` will return -1 in error cases.
Should handle the possible error scenarios (e.g., return error value, print error msgs, etc.) in the caller function and propagate the errors to upper layer.